--- gcc-3.3/libstdc++-v3/config/io/basic_file_stdio.cc.orig	2003-05-01 20:30:25.000000000 +0200
+++ gcc-3.3/libstdc++-v3/config/io/basic_file_stdio.cc	2003-05-17 09:37:01.000000000 +0200
@@ -239,7 +239,7 @@
   streamsize
   __basic_file<char>::showmanyc_helper()
   {
-#ifdef FIONREAD
+#if defined(FIONREAD) && (defined(_GLIBCPP_HAVE_SYS_IOCTL_H) || defined(_GLIBCPP_HAVE_SYS_FILIO_H))
     // Pipes and sockets.    
     int __num = 0;
     int __r = ioctl(this->fd(), FIONREAD, &__num);
