--- gcc-3.3.3/gcc/Makefile.in.orig	2004-02-16 18:57:42.824016056 +0100
+++ gcc-3.3.3/gcc/Makefile.in	2004-02-16 18:59:52.167352880 +0100
@@ -65,9 +65,9 @@
 # TCFLAGS is used for compilations with the GCC just built.
 XCFLAGS =
 TCFLAGS =
-CFLAGS = -g
-STAGE1_CFLAGS = -g @stage1_cflags@
-BOOT_CFLAGS = -g -O2
+CFLAGS =
+STAGE1_CFLAGS = @stage1_cflags@
+BOOT_CFLAGS = -O2
 
 # Flags to determine code coverage. When coverage is disabled, this will
 # contain the optimization flags, as you normally want code coverage
@@ -390,7 +390,7 @@
 
 # Options to use when compiling libgcc2.a.
 #
-LIBGCC2_DEBUG_CFLAGS = -g
+LIBGCC2_DEBUG_CFLAGS =
 LIBGCC2_CFLAGS = -O2 $(LIBGCC2_INCLUDES) $(GCC_CFLAGS) $(TARGET_LIBGCC2_CFLAGS) $(LIBGCC2_DEBUG_CFLAGS) $(GTHREAD_FLAGS) -DIN_LIBGCC2 -D__GCC_FLOAT_NOT_NEEDED @inhibit_libc@
 
 # Additional options to use when compiling libgcc2.a.
--- gcc-3.3.3/gcc/ada/Makefile.in.orig	2004-02-16 08:29:33.000000000 +0100
+++ gcc-3.3.3/gcc/ada/Makefile.in	2004-02-16 19:04:55.708207656 +0100
@@ -73,7 +73,7 @@
 # to the stage2 and stage3 compilations
 # XCFLAGS is used for most compilations but not when using the GCC just built.
 XCFLAGS =
-CFLAGS = -g
+CFLAGS =
 BOOT_CFLAGS = -O $(CFLAGS)
 # These exists to be overridden by the x-* and t-* files, respectively.
 X_CFLAGS =
@@ -127,7 +127,7 @@
 SOME_ADAFLAGS =-gnata
 FORCE_DEBUG_ADAFLAGS = -g
 GNATLIBFLAGS = -gnatpg
-GNATLIBCFLAGS = -g -O2
+GNATLIBCFLAGS = -O2
 GNATLIBCFLAGS_FOR_C = $(GNATLIBCFLAGS) $(TARGET_LIBGCC2_CFLAGS) -fexceptions \
 	-DIN_RTS
 ALL_ADA_CFLAGS = $(X_ADA_CFLAGS) $(T_ADA_CFLAGS) $(ADA_CFLAGS)
--- gcc-3.3.3/libjava/Makefile.am.orig	2004-02-16 08:29:34.000000000 +0100
+++ gcc-3.3.3/libjava/Makefile.am	2004-02-16 19:02:25.582030304 +0100
@@ -95,7 +95,7 @@
 AM_CFLAGS = @LIBGCJ_CFLAGS@
 endif
 
-JCFLAGS = -g
+JCFLAGS =
 JC1FLAGS = @LIBGCJ_JAVAFLAGS@ $(GCJFLAGS)
 
 LIBFFIINCS = @LIBFFIINCS@
--- gcc-3.3.3/libjava/Makefile.in.orig	2004-02-16 08:29:34.000000000 +0100
+++ gcc-3.3.3/libjava/Makefile.in	2004-02-16 19:02:31.196176824 +0100
@@ -180,7 +180,7 @@
 @USING_GCC_TRUE@AM_CFLAGS = @USING_GCC_TRUE@@LIBGCJ_CFLAGS@ $(WARNINGS)
 @USING_GCC_FALSE@AM_CFLAGS = @USING_GCC_FALSE@@LIBGCJ_CFLAGS@
 
-JCFLAGS = -g
+JCFLAGS =
 JC1FLAGS = @LIBGCJ_JAVAFLAGS@ $(GCJFLAGS)
 
 LIBFFIINCS = @LIBFFIINCS@
