--- trunk/gcc/Makefile.in.orig	2006-07-02 15:31:20.000000000 +0200
+++ trunk/gcc/Makefile.in	2006-07-02 20:55:33.573762250 +0200
@@ -151,9 +151,9 @@
 TCFLAGS =
 CFLAGS = @CFLAGS@
 LDFLAGS = @LDFLAGS@
-STAGE1_CFLAGS = -g @stage1_cflags@
+STAGE1_CFLAGS = @stage1_cflags@
 STAGE1_CHECKING = -DENABLE_CHECKING -DENABLE_ASSERT_CHECKING
-BOOT_CFLAGS = -g -O2
+BOOT_CFLAGS = -O2
 
 # Flags to determine code coverage. When coverage is disabled, this will
 # contain the optimization flags, as you normally want code coverage
@@ -547,7 +547,7 @@
 
 # Options to use when compiling libgcc2.a.
 #
-LIBGCC2_DEBUG_CFLAGS = -g
+LIBGCC2_DEBUG_CFLAGS =
 LIBGCC2_CFLAGS = -O2 $(LIBGCC2_INCLUDES) $(GCC_CFLAGS) $(TARGET_LIBGCC2_CFLAGS) \
 		 $(LIBGCC2_DEBUG_CFLAGS) $(GTHREAD_FLAGS) \
 		 -DIN_LIBGCC2 -D__GCC_FLOAT_NOT_NEEDED \
--- gcc-4.0-20040919/gcc/ada/Makefile.in.orig	2004-09-13 12:18:40.000000000 +0200
+++ gcc-4.0-20040919/gcc/ada/Makefile.in	2004-09-26 11:40:30.070894416 +0200
@@ -73,7 +73,7 @@
 # to the stage2 and stage3 compilations
 # XCFLAGS is used for most compilations but not when using the GCC just built.
 XCFLAGS =
-CFLAGS = -g
+CFLAGS =
 BOOT_CFLAGS = -O $(CFLAGS)
 # These exists to be overridden by the x-* and t-* files, respectively.
 X_CFLAGS =
@@ -118,9 +118,9 @@
 ADA_CFLAGS =
 ADAFLAGS = -W -Wall -gnatpg -gnata
 SOME_ADAFLAGS =-gnata
-FORCE_DEBUG_ADAFLAGS = -g
+FORCE_DEBUG_ADAFLAGS =
 GNATLIBFLAGS = -gnatpg -nostdinc
-GNATLIBCFLAGS = -g -O2
+GNATLIBCFLAGS = -O2
 GNATLIBCFLAGS_FOR_C = $(GNATLIBCFLAGS) $(TARGET_LIBGCC2_CFLAGS) -fexceptions \
 	-DIN_RTS
 ALL_ADA_CFLAGS = $(X_ADA_CFLAGS) $(T_ADA_CFLAGS) $(ADA_CFLAGS)
--- gcc/libada/Makefile.in.orig	2005-11-04 13:49:08.000000000 +0000
+++ gcc/libada/Makefile.in	2005-11-24 23:52:14.739531296 +0000
@@ -54,7 +54,6 @@
 include $(GCC_DIR)/libada-mk
 
 TARGET_LIBGCC2_CFLAGS=
-GNATLIBCFLAGS= -g -O2
 # Get target-specific overrides for TARGET_LIBGCC2_CFLAGS
 # and possibly GNATLIBCFLAGS.  Currently this uses files
 # in gcc/config.  The 'subst' call is used to rerelativize them
--- gcc-4.0-20040919/libffi/Makefile.am.orig	2004-08-30 17:42:59.000000000 +0200
+++ gcc-4.0-20040919/libffi/Makefile.am	2004-09-26 11:44:10.789340112 +0200
@@ -138,7 +138,7 @@
 libffi_convenience_la_SOURCES = $(libffi_la_SOURCES)
 nodist_libffi_convenience_la_SOURCES = $(nodist_libffi_la_SOURCES)
 
-AM_CFLAGS = -Wall -g -fexceptions
+AM_CFLAGS = -Wall -fexceptions
 
 libffi_la_LDFLAGS = -version-info `grep -v '^\#' $(srcdir)/libtool-version`
 
--- gcc-4.0-20040919/libffi/Makefile.in.orig	2004-09-19 19:47:04.000000000 +0200
+++ gcc-4.0-20040919/libffi/Makefile.in	2004-09-26 11:44:17.253357432 +0200
@@ -385,7 +385,7 @@
 nodist_libffi_la_SOURCES = $(am__append_1) $(am__append_2) $(am__append_3) $(am__append_4) $(am__append_5) $(am__append_6) $(am__append_7) $(am__append_8) $(am__append_9) $(am__append_10) $(am__append_11) $(am__append_12) $(am__append_13) $(am__append_14) $(am__append_15) $(am__append_16) $(am__append_17) $(am__append_18)
 libffi_convenience_la_SOURCES = $(libffi_la_SOURCES)
 nodist_libffi_convenience_la_SOURCES = $(nodist_libffi_la_SOURCES)
-AM_CFLAGS = -Wall -g -fexceptions
+AM_CFLAGS = -Wall -fexceptions
 libffi_la_LDFLAGS = -version-info `grep -v '^\#' $(srcdir)/libtool-version`
 AM_CPPFLAGS = -I. -I$(top_srcdir)/include -Iinclude -I$(top_srcdir)/src
 AM_CCASFLAGS = $(AM_CPPFLAGS)
--- gcc-4.0-20040919/libjava/Makefile.am.orig	2004-09-10 10:22:58.000000000 +0200
+++ gcc-4.0-20040919/libjava/Makefile.am	2004-09-26 11:46:24.041082768 +0200
@@ -179,7 +179,7 @@
 ## Extra CFLAGS used for JNI C sources shared with GNU Classpath.
 PEDANTIC_CFLAGS = -ansi -pedantic -Wall -Wno-long-long
 
-JCFLAGS = -g
+JCFLAGS =
 JC1FLAGS = @LIBGCJ_JAVAFLAGS@ $(GCJFLAGS)
 
 LIBFFIINCS = @LIBFFIINCS@
--- gcc-4.0-20040919/libjava/Makefile.in.orig	2004-09-19 19:47:04.000000000 +0200
+++ gcc-4.0-20040919/libjava/Makefile.in	2004-09-26 11:47:02.431246576 +0200
@@ -4167,7 +4167,7 @@
 @USING_GCC_FALSE@AM_CFLAGS = @LIBGCJ_CFLAGS@
 @USING_GCC_TRUE@AM_CFLAGS = @LIBGCJ_CFLAGS@ $(WARNINGS)
 PEDANTIC_CFLAGS = -ansi -pedantic -Wall -Wno-long-long
-JCFLAGS = -g
+JCFLAGS =
 JC1FLAGS = @LIBGCJ_JAVAFLAGS@ $(GCJFLAGS)
 AM_CPPFLAGS = -I$(top_srcdir) -Iinclude -I$(top_srcdir)/include \
 	$(GCINCS) $(THREADINCS) $(INCLTDL) \
