--- a/Makefile.in	2006-07-02 15:31:20.000000000 +0200
+++ b/Makefile.in	2006-07-02 20:55:33.573762250 +0200
@@ -274,7 +274,7 @@
 
 # Flags to pass to stage2 and later makes.  They are defined
 # here so that they can be overridden by Makefile fragments.
-BOOT_CFLAGS= -g -O2
+BOOT_CFLAGS= -O2
 BOOT_LDFLAGS=
 
 BISON = @BISON@
--- a/gcc/Makefile.in	2008-02-25 15:53:34.000000000 +0100
+++ b/gcc/Makefile.in	2008-03-03 16:42:29.000000000 +0100
@@ -560,7 +560,7 @@
 
 # Options to use when compiling libgcc2.a.
 #
-LIBGCC2_DEBUG_CFLAGS = -g
+LIBGCC2_DEBUG_CFLAGS =
 LIBGCC2_CFLAGS = -O2 $(LIBGCC2_INCLUDES) $(GCC_CFLAGS) $(TARGET_LIBGCC2_CFLAGS) \
 		 $(LIBGCC2_DEBUG_CFLAGS) $(GTHREAD_FLAGS) \
 		 -DIN_LIBGCC2 -D__GCC_FLOAT_NOT_NEEDED \
--- a/gcc/ada/gcc-interface/Makefile.in	2004-09-13 12:18:40.000000000 +0200
+++ b/gcc/ada/gcc-interface/Makefile.in	2004-09-26 11:40:30.070894416 +0200
@@ -66,7 +66,7 @@
 # CFLAGS is for the user to override to, e.g., do a bootstrap with -O2.
 # BOOT_CFLAGS is the value of CFLAGS to pass
 # to the stage2 and stage3 compilations
-CFLAGS = -g
+CFLAGS =
 BOOT_CFLAGS = -O $(CFLAGS)
 # These exists to be overridden by the t-* files, respectively.
 T_CFLAGS =
@@ -102,9 +102,9 @@
 ADA_CFLAGS =
 ADAFLAGS = -W -Wall -gnatpg -gnata
 SOME_ADAFLAGS =-gnata
-FORCE_DEBUG_ADAFLAGS = -g
+FORCE_DEBUG_ADAFLAGS =
 GNATLIBFLAGS = -gnatpg -nostdinc
-GNATLIBCFLAGS = -g -O2
+GNATLIBCFLAGS = -O2
 GNATLIBCFLAGS_FOR_C = $(GNATLIBCFLAGS) $(TARGET_LIBGCC2_CFLAGS) -fexceptions \
 	-DIN_RTS
 ALL_ADAFLAGS = $(CFLAGS) $(ADA_CFLAGS) $(ADAFLAGS)
--- a/libada/Makefile.in	2005-11-04 13:49:08.000000000 +0000
+++ b/libada/Makefile.in	2005-11-24 23:52:14.739531296 +0000
@@ -49,13 +49,13 @@
 LDFLAGS=
 
 # The tedious process of getting CFLAGS right.
-CFLAGS=-g
+CFLAGS=
 LOOSE_WARN = -W -Wall -Wwrite-strings -Wstrict-prototypes -Wmissing-prototypes
 GCC_WARN_CFLAGS = $(LOOSE_WARN)
 WARN_CFLAGS = @warn_cflags@
 
 TARGET_LIBGCC2_CFLAGS=
-GNATLIBCFLAGS= -g -O2
+GNATLIBCFLAGS= -O2
 
 # Get target-specific overrides for TARGET_LIBGCC2_CFLAGS.
 host_subdir = @host_subdir@
--- a/libffi/Makefile.in	2004-08-30 17:42:59.000000000 +0200
+++ b/libffi/Makefile.in	2004-09-26 11:44:10.789340112 +0200
@@ -453,7 +453,7 @@
 	$(am__append_21) $(am__append_22)
 libffi_convenience_la_SOURCES = $(libffi_la_SOURCES)
 nodist_libffi_convenience_la_SOURCES = $(nodist_libffi_la_SOURCES)
-AM_CFLAGS = -Wall -g -fexceptions
+AM_CFLAGS = -Wall -fexceptions
 LTLDFLAGS = $(shell $(SHELL) $(top_srcdir)/../libtool-ldflags $(LDFLAGS))
 libffi_la_LDFLAGS = -version-info `grep -v '^\#' $(srcdir)/libtool-version` $(LTLDFLAGS)
 AM_CPPFLAGS = -I. -I$(top_srcdir)/include -Iinclude -I$(top_srcdir)/src
--- a/libffi/Makefile.am	2008-05-09 12:10:53.000000000 +0200
+++ b/libffi/Makefile.am	2008-11-28 02:02:01.690063811 +0100
@@ -154,7 +154,7 @@
 libffi_convenience_la_SOURCES = $(libffi_la_SOURCES)
 nodist_libffi_convenience_la_SOURCES = $(nodist_libffi_la_SOURCES)
 
-AM_CFLAGS = -Wall -g -fexceptions
+AM_CFLAGS = -Wall -fexceptions
 
 LTLDFLAGS = $(shell $(SHELL) $(top_srcdir)/../libtool-ldflags $(LDFLAGS))
 
--- a/libjava/Makefile.am	2004-09-10 10:22:58.000000000 +0200
+++ b/libjava/Makefile.am	2004-09-26 11:46:24.041082768 +0200
@@ -179,7 +179,7 @@
 ## Extra CFLAGS used for JNI C sources shared with GNU Classpath.
 PEDANTIC_CFLAGS = -ansi -pedantic -Wall -Wno-long-long
 
-JCFLAGS = -g
+JCFLAGS =
 JC1FLAGS = @LIBGCJ_JAVAFLAGS@ $(GCJFLAGS)
 
 LIBFFIINCS = @LIBFFIINCS@
--- a/libjava/Makefile.in	2004-09-19 19:47:04.000000000 +0200
+++ b/libjava/Makefile.in	2004-09-26 11:47:02.431246576 +0200
@@ -4167,7 +4167,7 @@
 @USING_GCC_FALSE@AM_CFLAGS = @LIBGCJ_CFLAGS@
 @USING_GCC_TRUE@AM_CFLAGS = @LIBGCJ_CFLAGS@ $(WARNINGS)
 PEDANTIC_CFLAGS = -ansi -pedantic -Wall -Wno-long-long
-JCFLAGS = -g
+JCFLAGS =
 JC1FLAGS = @LIBGCJ_JAVAFLAGS@ $(GCJFLAGS)
 AM_CPPFLAGS = -I$(top_srcdir) -Iinclude -I$(top_srcdir)/include \
 	$(GCINCS) $(THREADINCS) $(INCLTDL) \
